src/state/reducers/worker.action-creator.ts

```diff
@@ -156,24 +202,6 @@ export class WorkerActionCreator {
         ...new Array(today.getDate() - 1).fill(ShiftCode.NZ)
       );
     }
-
-    updatedSchedule.shifts = {
```

Comment:
@pectom 
Było dwie podobne metody, więc wyniosłem wspólną cześć do osobnej, ponieważ musiałem wprowadzić trochę zmian do tej logiki, i żeby nie potrzeba było tego robić w dwóch miejscach

Reply:
Taa, to było rozwiązanie na szybko przed prezentacją. Super, że to poprawiłeś :D

Reply:
Żaden problem ;)